packages/keystore/lib/db/migrate.ts

```diff
@@ -0,0 +1,32 @@
+import path from 'node:path';
```

Comment:
I understand why you created a dedicated file, however, it's going to create two lock tables for each package that uses this pattern. I don't have a strong opinion because it's definitely handy to split everything, just not sure it's worth it imo

Reply:
Yeah, the main difference is that it's not in the same schema. Let's discuss tomorrow ☺️